jjb/jobs/polarion-test-case-importer.yaml

```diff
@@ -0,0 +1,76 @@
+- job:
+    name: 'polarion-test-case-importer'
+    node: 'f25-np'
+    triggers:
+        - timed: '@midnight'
+    scm:
+        - git:
+            url: https://github.com/quipucords/camayoc.git
+            branches:
+                - '*/master'
+            skip-tag: true
+    wrappers:
+        - inject-passwords:
+            global: true
+            mask-password-params: true
+        - ssh-agent-credentials:
+            users:
+                - '390bdc1f-73c6-457e-81de-9e794478e0e'
+    builders:
+        - shining-panda:
+            build-environment: virtualenv
+            python-version: System-CPython-2.7
+            clear: true
+            nature: shell
+            command: |
+                rm -rf pylarion
+
```

Comment:
Have you considered just trashing the whole workspace each time?

Reply:
https://docs.openstack.org/infra/jenkins-job-builder/wrappers.html#wrappers.workspace-cleanup%20(pre-build)

Reply:
Since we are using nodepool the workspace should be already empty because we get one slave per run and that slave has no workspace created when provisioned.

The remove is just a safety if for some reason Jenkins uses the same slave and because I am not letting the git plugin clone the repository since I need to get it from an env var. Used `-f` to ensure rm won't fail if the directory is not present.

Reply:
Ok sounds good :D